### PR TITLE
[WIP]Add create accout screen

### PR DESCRIPTION
**Pull request by alexcherman**

### CODE REVIEW

File: universal-login-react/src/ui/styles/topUpModalDefaults.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.universal-login-topup.topup {
+  width: 570px;
+  max-width: 100%;
+  margin: 0 auto;
+}
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/styles/topUpModalDefaults.css]: newline

File: universal-login-react/src/ui/TopUp/TopUp.tsx [hunk 2]
```diff
@@ -14,7 +15,11 @@ interface TopUpProps {
 export const TopUp = ({contractAddress, startModal, onRampConfig}: TopUpProps) => {
   const [modal, setModal] = useState<TopUpComponentType>(startModal || TopUpComponentType.choose);
   if (modal === TopUpComponentType.choose) {
-    return (<TopUpChoose onMethodChoose={setModal}/>);
+    return (
+      <ModalWrapper isVisible modalPosition="bottom">
```
Comment (xitronix) [inline universal-login-react/src/ui/TopUp/TopUp.tsx]: Position should be got from props and have default value 'bottom'?

File: universal-login-wallet/src/ui/react/App.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import SettingsScreen from './Settings/SettingsScreen';
 import {useServices} from '../hooks';
 import {WelcomeScreen} from './Home/WelcomeScreen';
 import { TermsAndConditionsScreen } from './Home/TermsAndConditionsScreen';
+import { CreateAccount } from './CreateAccount/CreateAccount';
```
Comment (xitronix) [inline universal-login-wallet/src/ui/react/App.tsx]: Extra spaces

File: universal-login-react/src/ui/TopUp/TopUp.tsx (hunk 2 above)
Comment (marekkirejczyk) [reply universal-login-react/src/ui/TopUp/TopUp.tsx]: Can we move it to custom css? #NextPr

**Pull request merged**